tools/vigiechiro/BilanEnrichiPFen.xml

```diff
@@ -0,0 +1,69 @@
+<tool id="vigiechiro_BilanEnrichiPF" name="Advanced restitution: 'Point fixe' protocol" version="@VERSION@">
+    <description>from Animal Detection on Acoustic Recordings</description>
+    <macros>
+        <import>vigiechiro_macros.xml</import>
+    </macros>   
+
+    <expand macro="vigiechiro_bilanPF_bilanRP_requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript 
```

Comment:
can you break this down to multiple lines to make it better readable?

Reply:
Do you mean create 2 macros ? 
``vigiechiro_bilanRP_requirements``
&
``vigiechiro_bilanPF_requirements``

Reply:
No something like this:
```
'$__tool_directory__/BilanEnrichiPF.R' 
     '$idc2' '$__tool_directory__/refPF.csv'
     '$__tool_directory__/SpeciesList.csv' 
&&
```

Reply:
Oh yes sure